lib/web_of_science/client.rb

```diff
@@ -13,9 +13,7 @@ class Client
     AUTH_WSDL = 'http://search.webofknowledge.com/esti/wokmws/ws/WOKMWSAuthenticate?wsdl'.freeze
     SEARCH_WSDL = 'http://search.webofknowledge.com/esti/wokmws/ws/WokSearch?wsdl'.freeze
 
-    delegate :logger, to: :WebOfScience
-
-    def initialize(auth_code, log_level = Settings.WOS.LOG_LEVEL.to_sym)
```

Comment:
For the changes you want, I don't see how it make sense to cut out the `Settings`.  Seems like you should just be overriding the setting to accomplish your debugging, not making it irrelevant, like:
```ruby
$ SETTINGS__WOS__LOG_LEVEL=error bundle exec rails c
Loading development environment (Rails 4.2.10)
[1] pry(main)> Settings.WOS.LOG_LEVEL
=> "error"
...
$ SETTINGS__WOS__LOG_LEVEL=warn bundle exec rails c
Loading development environment (Rails 4.2.10)
[1] pry(main)> Settings.WOS.LOG_LEVEL
=> "warn"
```

Reply:
That's an interesting idea, thanks.  But, I think this change will be useful in general.  All the savon log info is really not informative in the main harvesting logs.  If you get a chance to look at the logs without this change, you can see why this change is generally useful.  At the level of the savon client, there's not much app-specific logging that we need an app-specific setting for it.  But, if we really should keep the log level, that could stay, so long as the client log file is separated out as in this PR.

Reply:
OK, let's keep it.  Pushed.